frontend/app/src/components/display/StatCard.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <v-card class="stat-card">
     <v-card-title>
-      {{ title }}
+      <slot name="title">{{ title }}</slot>
```

Comment:
are you passing the title as a template somewhere?

Reply:
Not anymore, in one of the iterations. I'll revert.